cmd/root.go

```diff
@@ -124,6 +131,9 @@ func init() {
 	RootCmd.PersistentFlags().String("discovery-server", "discover.thethingsnetwork.org:1900", "The address of the Discovery server")
 	viper.BindPFlag("discovery-server", RootCmd.PersistentFlags().Lookup("discovery-server"))
 
+	RootCmd.PersistentFlags().String("noc-server", "noc.thethingsnetwork.org:1905", "The address of the Noc server")
```

Comment:
1. of the **NOC** server
2. default should be `""` as NOC participation is opt-in